packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1023,12 +1094,7 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
     }
 
     private get summaryConfiguration() {
-        return {
-            // the defaults
-            ... DefaultSummaryConfiguration,
-            // the runtime configuration overrides
-            ... this.runtimeOptions.summaryOptions?.summaryConfigOverrides,
-        };
+        return this.currentSummaryConfiguration;
     }
```

Comment:
I'd get rid of getter / inline usage - there is not much value in having 1-liners like that